spec.bs

```diff
@@ -767,6 +772,12 @@ The user agent MAY utilize the information in
 into a language and using locale-based formatting consistent with that of the
 website.
 
+If {{SecurePaymentConfirmationRequest/showOptOut}} is `true`, the user agent
+MUST provide an additional button for the user to indicate that they want to
+opt out of the process. If this button is selected, then the user agent must
```

Comment:
Here, instead of 'button is selected', we might use language something like: "If the user indicates that they wish to opt-out, the user agent must ..."

Reply:
Done